azure-pipelines.yml

```diff
@@ -81,11 +82,12 @@ jobs:
         parameters:
           appHostGitPath: git://$(AppHostingSdkGitPath3)@$(AppHostingSdkGitRef)
 
-      - task: Yarn@2
-        displayName: 'Build Test App Local'
+      - task: CmdLine@2
+        displayName: 'Build Test App local'
```

Comment:
nit: I feel "local" should remain capitalized

Reply:
Resolved in latest commit.